indexer/src/view/infinite.ts

```diff
@@ -0,0 +1,43 @@
+import {Index, PrimaryColumn, ViewColumn, ViewEntity} from "typeorm";
+
+@ViewEntity({
+    expression: `
+        select "txHash"                              "transactionHash",
+               "name"                                "transactionType",
+               (content -> 'user_id')::numeric       "transactionOwner",
+               (content -> 'generation')::numeric    "gameGeneration",
+               (content -> 'state')::numeric         "gameState",
+               (content -> 'cell_index')::numeric    "revivedCellIndex",
```

Comment:
I think we don't necesseraily need to have `cell_index` here, right? What happens if we don't have it?

Reply:
It holds the index of a cell user gave life to. Without it we would not hold the information about reviving cells.

Reply:
I mean what if we don't have `cell_index` in json here and we want to treat it as numeric?

Reply:
We just get `null`